docs/faq.rst

```diff
@@ -124,10 +124,28 @@ Which ML libraries does Fairlearn support?
     `predict()` methods. Also, any classification or regression
     algorithm can be evaluated using our metrics.
 
+Does Fairlearn support multi-class classification?
```

Comment:
I think that there should be 'metrics' and 'mitigation' portions to this FAQ.

Reply:
Indeed.